docs/source/notebooks/moment_interaction.ipynb

```diff
@@ -282,24 +282,55 @@
+    "\n",
+    "5. ``kappa_0`` - zero curvature compression, this must be at start of the list as it results in the largest compression force. Note that the second value in the tuple is not used and can be anything.<br>\n",
+    "\n",
+    "The following parameters can be used to modify the moment interaction diagram:\n",
+    "\n",
+    "1. ``limits`` - List of control points that define the start and end of the interaction diagram. List length must equal two. The default limits range from decompression ``(\"D\", 1.0)`` to zero curvature tension ``(\"d_n\", 1e-6)``.\n",
+    "\n",
+    "2. ``control_points`` - adding control points to the moment interaction diagram allows the user to ensure certain points are captured on the moment interaction diagram.  The default argument for ``control_points`` is ``[(\"kappa0\", 0.0), (\"fy\", 1.0), (\"N\", 0.0)]`` which produces control points at the pure compression point, the balanced point and the pure bending point. \n",
+    "    \n",
+    "\n",
+    "3. ``labels`` - A list of labels to apply to the ``limits`` and ``control_points`` for plotting purposes. The first two values in ``labels`` apply labels to the ``limits``, the remaining values apply labels to the ``control_points``.\n",
+    "\n",
+    "4. ``n_points`` - The number of points to compute including and between the ``limits`` of the moment interaction diagram.\n",
+    "\n",
+    "5. ``n_spacing`` - If provided, overrides ``n_points`` and spaces the points on the moment interaction diagram by an axial force ``n_spacing``.\n",
+    "\n",
+    "6. ``max_comp`` - Limits the maximum compressive force in the moment interaction diagram by cutting off the top of the diagram."
    ]
   },
   {
    "cell_type": "markdown",
    "id": "303c4420",
    "metadata": {},
    "source": [
-    "The below example will use all these features, creating additional control points at zero strain, 50% strain and a neutral axis depth of 200 mm. Note, care must be taken to ensure that the ``control_points`` are ordered from largest neutral axis depth to smallest neutral axis depth otherwise a ``ValueError`` will be raised. Labels are applied and the number of points between each control point is modified. A maximum compressive force of 6800 kN is provided."
+    "The below example will use these features, creating additional control points at:\n",
+    "\n",
+    "- Decompression of the concrete ``(\"D\", 1.0)`` - label ``C``\n",
+    "- Decompression of the steel ``(\"fy\", 0.0)`` - label ``D``\n",
+    "- 50% yield strain in the steel ``(\"fy\", 0.5)`` - label ``E``\n",
+    "- 100% yield strain in the steel ``(\"fy\", 1.0)`` - label ``F``\n",
+    "- Neutral axis depth of 200 mm ``(\"d_n\", 200.0)`` - label ``G``\n",
+    "\n",
+    "A maximum compressive force of 6800 kN is provided. Label ``A`` is placed at zero axial force and label ``B`` is placed at the intersection with the moment interaction diagram.\n",
```

Comment:
Label point A is at zero moment, not zero axial load I think.

Reply:
Yes good pickup